conf/application.conf

```diff
@@ -150,10 +132,9 @@ tracking-consent-frontend {
 upscan {
   callbackBaseUrl = "http://localhost:12600"
   minFileSize = "1b"
-  maxFileSize = "5000000b"
   maxFiles    = 5
+  maxFileSize = "5000000b"
 }
 
 internal-auth.token = "82fac546-6de3-47bb-a1d7-357d68ecd3ac"
 object-store.default-retention-period = 1-month
-create-internal-auth-token-on-start = true
```

Comment:
Why are we removing this?

Reply:
I'll add this back in as it's used and got accidentally removed.